external/storm-solr/src/main/java/org/apache/storm/solr/bolt/SolrUpdateBolt.java

```diff
@@ -74,6 +75,9 @@ public SolrUpdateBolt(SolrConfig solrConfig, SolrMapper solrMapper, SolrCommitSt
     @Override
     public void prepare(Map stormConf, TopologyContext context, OutputCollector collector) {
         this.collector = collector;
+        solrMapper.configure();
+        if (solrConfig.enableKerberos())
```

Comment:
This should be moved to above `solrMapper.configure()` so that it does not need to be set in that code path.